playground/AzureStorageEndToEnd/AzureStorageEndToEnd.AppHost/aspire-dashboard-containerapp.bicep

```diff
@@ -0,0 +1,52 @@
+param location string
```

Comment:
This file shouldn't be getting emitted. You need to check whether the manifest publishing callback is set to ignore.

Reply:
Yep